src/main/io/gps.c

```diff
@@ -789,6 +797,19 @@ static bool gpsNewFrameNMEA(char c)
                         *gpsPacketLogChar = LOG_NMEA_RMC;
                         gpsSol.groundSpeed = gps_Msg.speed;
                         gpsSol.groundCourse = gps_Msg.ground_course;
+                        // This check will miss 00:00:00.00, but we shouldn't care - next report will be valid
+                        if(!rtcHasTime() && gps_Msg.date != 0 && gps_Msg.time != 0)
```

Comment:
Why do we need to check `gps_Msg.time != 0` here?

Reply:
It isn't needed, only checking the date is sufficient. This was just how it was in inav.

Reply:
Ok. Keeping it the same makes sense then. (Also, not sure if it's possible that some units are sending the date first, then the time, so this could at least theoretically result in the time being set as the current date, 00:00:00.)

Reply:
I added this again in the last commit.